.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -13,8 +13,9 @@ Your issue will be triaged by the RNW team according to this process: https://gi
 ## Environment
 Run the following in your terminal and copy the results here.
 1. `npx react-native --version`:
-2. `npx react-native run-windows --info`:
-3. `reg query "HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\AppModelUnlock"`
+2. `npx react-native info`:
+3. `npx react-native run-windows --info`:
+4. `reg query "HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\AppModelUnlock"`
```

Comment:
```suggestion
```

We can get rid of these lines now because react-native info will show the same info :)
We can remove --info from run-windows in a subsequent PR when we're sure everyone has the updated version of react-native